### DISCUSSION TITLE
write @livewireStyles another method

### DISCUSSION THREAD

**Discussion by vahidalvandi** [livewire/livewire#5048] [1. Help]
he , i want to run @livewireStyles  or @livewireScripts to another method
becuase i want to convert to blade file to compiled format php 

same of synatx after compile convert to    
 `@stack('styles')   `
  `  <?php echo $__env->yieldPushContent('styles'); ?>`
but `@livewireStyles`  not convert

**Comment by michael-rubel** [comment]


**Reply by vahidalvandi** [reply]
nice worked
